test/Datadog.Trace.ClrProfiler.IntegrationTests/SqlServerTests.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using MessagePack;
+using Xunit;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class SqlServerTests : TestHelper
+    {
+        [Fact]
+        [Trait("Category", "EndToEnd")]
+        public void SubmitsTraces()
+        {
+            using (var agent = new MockTracerAgent())
+            {
+                using (var process = StartSample("SqlServer"))
```

Comment:
Nice how this ends up being such a short test after all the framework is in place 👍